src/data/things/album.js

```diff
@@ -86,6 +86,7 @@ export class Album extends Thing {
       }),
     ],
 
+    alwaysReferenceByDirectory: flag(false),
```

Comment:
Noting that `alwaysReferenceByDirectory` is implemented in a more complex way for tracks, but that's only to handle the special it's-implied behavior for rereleases. A flag works just fine here 👍